mmv1/third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -5236,6 +5275,42 @@ resource "google_container_cluster" "with_resource_labels" {
 `, location)
 }
 
+
+func testAccContainerCluster_withExternalIpsConfigEnabled(projectID string, clusterName string) string {
+	return fmt.Sprintf(`
+	data "google_project" "project" {
+		project_id = "%s"
+	}
+
+	resource "google_container_cluster" "with_external_ips_config" {
+		name               = "%s"
+		location           = "us-central1-a"
+		initial_node_count = 1
+		remove_default_node_pool = true
```

Comment:
This line is causing your test to fail. it is a virtual field (isn't actually returned by the API). In the import step, the test runner expected this field to be set and returned by the API but it wasn't. Normally you'd have to set the field to be ignored, but if you just remove this whole function (see my comment above), that should also fix

Reply:
Fixed